tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -147,3 +149,48 @@ def test_workload_rbd_cephfs(
     """
     logger.info(workload_storageutilization_50p_rbd)
     logger.info(workload_storageutilization_50p_cephfs)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-2125")
+def test_workload_rbd_cephfs_minimal(
+    workload_storageutilization_05p_rbd,
+    workload_storageutilization_05p_cephfs
+):
+    """
+    Similar to test_workload_rbd_cephfs, but using only 5% of total OCS
+    capacity. This still test the workload, but it's bit faster and (hopefully)
+    without big impact on the cluster itself.
+
+    In this test we are only checking whether the storage utilization workload
+    failed or not. The main point of having this included in tier1 suite is to
+    see whether we are able to actually run the fio write workload without any
+    direct failure (fio job could fail to be scheduled, fail during writing or
+    timeout when write progress is too slow ...).
+
+    Please note that reaching 5% of total OCS capacity means that workload
+    fixtures specified above will try to write data based on current cluster
+    wide storage utilization to meet the specified target. If the current
+    cluster utilization is already above 5%, this will fail. This is targeted
+    to a fresh just installed CI cluster.
+    """
+    logger.info("checking fio report results as provided by workload fixtures")
+    msg = "workload results were recorded and provided to the test"
```

Comment:
I don't think that this message is informative in case that there are no results in workload fixtures.

Reply:
For assert message I usually just translate the condition into a natural language. And I recall that there was a feedback about that in the past. Would using *should* instead of *were* fix this? I personally find negating the statement confusing, but maybe there are guidelines about this.

Reply:
fixed in ca88d9d658a804c04a9938c6a3a4705ed24680bf

Reply:
I assume that either negating the message or using should would improve understandability of the error if the error occurs.